tools/cldr-apps/js/src/esm/cldrProgress.js

```diff
@@ -18,30 +19,62 @@ const USE_NEW_PROGRESS_WIDGET = false;
 let progressWrapper = null;
 
 let sectionProgressStats = null;
+let voterProgressStats = null;
+let localeProgressStats = null;
+
+let sectionProgressRows = null;
+
+class MeterData {
+  constructor(description, votes, total, level) {
+    this.description = description;
+    this.votes = votes;
+    this.total = total;
+    this.level = level;
+    if (!description) {
+      this.percent = 0;
+      this.title = "No data";
+      return;
+    }
+    this.percent = friendlyPercent(votes, total);
+    const locale = cldrStatus.getCurrentLocale();
+    const localeName = cldrLoad.getLocaleName(locale);
+    this.title =
```

Comment:
I'd probably do this text calculation closer to the glass (such as a `ref`  in a Vue template ) so that it can be [reactive](https://v3.vuejs.org/guide/reactivity-fundamentals.html), but perhaps this is done as a class so that it can plug in to the `<a-progress`.

Reply:
Whenever cldrProgress calls updateSectionMeter, updateVoterMeter, or updateLocaleMeter, the data in ProgressMeters.vue gets updated and displayed -- so it is _reactive_, as I understand that word.

I can't say that I fully understand https://v3.vuejs.org/guide/reactivity-fundamentals.html or when/why it would be appropriate to use the "reactive" keyword to "create a reactive state from a JavaScript object". I don't think anything like that is necessary in this case.